lib/ssl/doc/src/ssl.xml

```diff
@@ -1006,6 +1006,29 @@ fun(srp, Username :: string(), UserState :: term()) ->
 	</desc>
       </datatype>
 
+      <datatype>
+	<name name="ocsp_stapling"/>
+	<desc><p>If true, OCSP stapling will be enabled, an extension of type "status_request" will be
+  included in the client hello to indicate the desire to receive certificate status information.
+  If false (the default), OCSP stapling will be disabled.</p>
+	</desc>
+      </datatype>
+
+      <datatype>
+	<name name="ocsp_responder_certs"/>
+	<desc><p>This option is a list of DER encoded certificates of OCSP responders which the client
+  trusts. This option is an empty list by default which means that the responders are implicitly
+  known to the server.</p>
```

Comment:
We should probably document here how the `ocsp_reponder_certs` option interacts with the list of `certs` received in `BasicOCSPResponse`. We should also consider if this option should take the value of `cacerts`/`cacertfile` if it is not set.

Reply:
> We should probably document here how the `ocsp_reponder_certs` option interacts with the list of `certs` received in `BasicOCSPResponse`. We should also consider if this option should take the value of `cacerts`/`cacertfile` if it is not set.

partially done, since I have checked the reply seems only cert is provided in the certs field.